src/Facebook/FacebookClient.php

```diff
@@ -203,25 +193,18 @@ public function sendRequest(FacebookRequest $request)
 
         list($url, $method, $headers, $body) = $this->prepareRequestMessage($request);
 
-        // Since file uploads can take a while, we need to give more time for uploads
-        $timeOut = static::DEFAULT_REQUEST_TIMEOUT;
-        if ($request->containsFileUploads()) {
-            $timeOut = static::DEFAULT_FILE_UPLOAD_REQUEST_TIMEOUT;
-        } elseif ($request->containsVideoUploads()) {
-            $timeOut = static::DEFAULT_VIDEO_UPLOAD_REQUEST_TIMEOUT;
-        }
```

Comment:
Yes I know. There is no way of us configure the client. This is the responsibility of the application author. He/She would change this setting if he/she notice that the file uploads are "time outing".